### PR TITLE
Debug curve plot

### PR DESCRIPTION
**Pull request by d-morrison**
fixes https://github.com/UCD-SERG/shiny_decay_curve/issues/4 by handling  r = 1 case specially; 
Pasted Graphic
is undefined when r = 1, but the limit is $y(t) = y_1 e^{-v(t-t_1)}$

### CODE REVIEW

**Comment by d-morrison** [comment]
@sschildhauer please review this PR; this is a good opportunity to get you up to speed on github code review

File: R/ab0.R [hunk 1]
```diff
@@ -1,19 +1,26 @@
 # uses r > 1 scale for shape
-ab0 <- function(
-    t,
-    curve_params) {
+ab0 <- function(t, curve_params) {
```
Comment (d-morrison) [inline R/ab0.R:2]: lint

File: R/ab0.R [hunk 2]
```diff
@@ -1,19 +1,26 @@
...
   y0 <- curve_params[["y0"]]
   y1 <- curve_params[["y1"]]
   t1 <- curve_params[["t1"]]
   alpha <- curve_params[["alpha"]]
-  shape <- curve_params[["r"]]
+  r <- curve_params[["r"]]
```
Comment (d-morrison) [inline R/ab0.R:7]: shortening varname to make lines below fit in less than 80 characters

File: R/ab0.R [hunk 3]
```diff
@@ -1,19 +1,26 @@
...
 
   beta <- bt(y0, y1, t1)
 
   yt <- 0
 
   yt_phase_1 <- y0 * exp(beta * t)
-  yt_phase_2 <- (y1^(1 - shape) - (1 - shape) * alpha * (t - t1))^(1 / (1 - shape))
+  if (r == 1) {
```
Comment (d-morrison) [inline R/ab0.R:14]: if r == 1, the usual expression for this function reduces to `0^Inf = 0`, which isn't correct; as r -> 1, the function should reduce to the exponential.

**Review by kristinawlai — APPROVED** [review]

**Pull request merged**